src/js/server/devpmt-express.js

```diff
@@ -0,0 +1,528 @@
+                        funcName: "fluid.stringTemplate",
+                        args: ["%prefsServerURL/preferences", {
+                            prefsServerURL: "{gpii.devpmt}.options.prefsServerURL"
+                        }]
+                    }
+                },
+                writable: true,
+                writeMethod: "POST"
+            }
+        },
+        prefsSafeKeyCreationDataSource: {
+            type: "kettle.dataSource.URL",
+            options: {
+                url: {
+                    expander: {
+                        funcName: "fluid.stringTemplate",
+                        args: ["%prefsServerURL/prefssafe-key-create", {
+                            prefsServerURL: "{gpii.devpmt}.options.prefsServerURL"
+                        }]
+                    }
+                },
+                writable: true,
+                writeMethod: "POST"
+            }
+        },
+        prefsSafeByGpiiKeyDataSource: {
+            type: "kettle.dataSource.URL",
+            options: {
+                url: {
+                    expander: {
+                        funcName: "fluid.stringTemplate",
+                        args: ["%prefsServerURL/prefssafe/%prefsSafeId", {
+                            prefsServerURL: "{gpii.devpmt}.options.prefsServerURL"
+                        }]
+                    }
+                },
+                termMap: {
+                    prefsSafeId: "%prefsSafeId"
+                }
+            }
+        },
+        cloudSafeCredCreateDataSource: {
+            type: "kettle.dataSource.URL",
+            options: {
+                url: {
+                    expander: {
+                        funcName: "fluid.stringTemplate",
+                        args: ["%prefsServerURL/add-cloud-credentials/%prefsSafeId", {
+                            prefsServerURL: "{gpii.devpmt}.options.prefsServerURL"
+                        }]
+                    }
+                },
+                termMap: {
+                    prefsSafeId: "%prefsSafeId"
+                },
+                writable: true,
+                writeMethod: "PUT"
+            }
+        },
+        cloudSafeUnlockDataSource: {
+            type: "kettle.dataSource.URL",
+            options: {
+                url: {
+                    expander: {
+                        funcName: "fluid.stringTemplate",
+                        args: ["%prefsServerURL/unlock-cloud-safe", {
+                            prefsServerURL: "{gpii.devpmt}.options.prefsServerURL"
+                        }]
+                    }
+                },
+                writable: true,
+                writeMethod: "POST"
+            }
+        }
+    },
+    invokers: {
+        stop: {
+            func: "{that}.destroy"
+            // funcName: "gpii.express.stopServer",
+            // args: ["that"]
+        }
+    }
+});
+
+gpii.devpmt.initialize = function (that) {
+    fluid.each(that.options.osList, function (osId) {
```

Comment:
This is more like an expander that produces an initial model value rather than an initialize function

Reply:
I've cleaned this up a small bit, removed part of it that wasn't needed anymore, and named it appropriately.